manifest_schema.json

```diff
@@ -29,162 +24,184 @@
     "pages"
   ],
   "properties": {
+    // Inherited from [[APPMANIFEST]]
```

Comment:
I don't think comments of the form `//…` or `/* … */` are allowed in JSON...

Reply:
Yes! Thanks, @xfq. Solved at 1694b72